obs-packaging/wait-obs.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+handle_error() {
+	local exit_code="${?}"
+	local line_number="${1:-}"
+	echo "Failed at $line_number: ${BASH_COMMAND}"
+	exit "${exit_code}"
+}
+trap 'handle_error $LINENO' ERR
+
+script_dir=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+
+run_in_docker() {
+	if [ -n "${USE_DOCKER:-}" ]; then
+		# shellcheck source=scripts/obs-docker.sh
```

Comment:
nit - commented out debug line?

Reply:
I use `shellcheck`  as part of my vim script linters so it marks as warnings, not needed not needed all all but make it a bit better for shellcheck users